libff/algebra/field_utils/bigint.hpp

```diff
@@ -1,6 +1,10 @@
 /** @file
  *****************************************************************************
  Declaration of bigint wrapper class around GMP's MPZ long integers.
+
+ Notice that this class has no arithmetic operators. This is deliberate. All
+ bigints should either be hardcoded or operated on the bit level to ensure
+ high performance.
```

Comment:
This comment is very helpful for giving context regarding the file! Thanks for adding it